Source/Scene/Primitive.js

```diff
@@ -1136,13 +1136,9 @@ function depthClampVS(vertexShaderSource) {
   // It is not to workaround an issue with Edge:
   //     https://developer.microsoft.com/en-us/microsoft-edge/platform/issues/12120362/
```

Comment:
Is this comment still relevant?

Reply:
nope, the link is dead and edge is based on chromium now

Reply:
nope, removed.